### ISSUE TITLE
Release 114.0

### ISSUE THREAD

**Issue by andyws** [hmrc/self-assessment-api#420]
I am assuming that the RAML contained in 114.0 is complete and accurate with respect to API released as 114.0.  Is this correct?

Is 114.0 available in all environments?

**Comment by rakeshavasarala** [comment]
0.114.0 is a Pre-Release. It is not available is any of the public facing environments.

Versions in Sandbox and Production.
Sandbox - 0.112.0
Production - 0.100.1

**Comment by andyws** [comment]
@rakeshavasarala 

Ok.  I note the label has been changed from Release to Pre-Release

Thank you for listing the versions in the other environments.  I was surprised to see 112 is in the sandbox.  How would I have found out this fact?

**Comment by rakeshavasarala** [comment]
I changed 0.114.0 to Pre-Release within 2 minutes of it being visible of UI. You were quicker.

In the latest release to Sandbox i.e version 0.112.0, we have added a section **Changelog** (SDST sent comms to all Developers using Sandbox) which points to the CHANGELOG.md file, but this is for production. There is a link to CHANGELOG-ET.md file which is for Sandbox.

We understand that this is not ideal and you would prefer to see the release version/changelog in the documentation.
